src/config/select.js

```diff
@@ -1,4 +1,3 @@
-import React from 'react';
```

Comment:
I couldn't find any notes that they will move React.createElement or the React.DOM helpers to the 'react-dom' package.